src/checks.rs

```diff
@@ -2533,6 +2538,7 @@ impl CheckKind {
                 format!("Use `{}` instead of `type(...)`", primitive.builtin())
             }
             CheckKind::UselessMetaclassType => "`__metaclass__ = type` is implied".to_string(),
+            CheckKind::TypingTextStrAlias => "`typing.Text` is depreciated use `str`".to_string(),
```

Comment:
Nit: mind changing `depreciated use str` to `deprecated, use str instead to match  the format of `DeprecatedUnittestAlias` below?